save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -736,18 +736,27 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                 }
 
                 element.files!!.asList().forEach { file ->
-                    val response: FileInfo = post(
+                    val response: FileInfoDto = post(
                         "$apiUrl/files/upload",
```

Comment:
Here we can use some flag, which will responsible of returning value `FileInfo` or `FileInfoDto`

Reply:
Added